eks-aggregator/helm/datadog.yaml

```diff
@@ -13,6 +13,7 @@ agents:
   useConfigMap: true
   customAgentConfig:
     logs_config:
-      logs_dd_url: "vector-haproxy.vector.svc.cluster.local:8080"
+      logs_dd_url: "internal-k8s-vector-vector-e67b4af534-965797666.us-east-1.elb.amazonaws.com:8080"
```

Comment:
🤔 sounds really specific, won't that change after being redeployed?

Reply:
Looking at the most recent commit, that's a generic hostname intended to be replaced with the `--set` option 🤔